deploy/helm/sumologic/Chart.yaml

```diff
@@ -13,7 +13,7 @@ sources:
   - https://github.com/SumoLogic/sumologic-kubernetes-collection
 dependencies:
   - name: fluent-bit
-    version: 0.15.4
+    version: 0.11.0
```

Comment:
Please upgrade the README support matrix as well.

Reply:
Actually we can use `0.12.1`- it also uses `1.6.10`

https://github.com/fluent/helm-charts/blob/fluent-bit-0.12.1/charts/fluent-bit/Chart.yaml

Reply:
changed to `0.12.1`